eng/scripts/Push-TestProxy.ps1

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+param(
+    [string] $AssetsJsonPath
+)
+
+function AssertInstalled() {
+    if (Get-Command test-proxy -ErrorAction Ignore) {
+        return $true;
+    }
+    $run = Read-Host "The test-proxy tool could not be found.`nWould you like to install it? [y/n]"
+    if ($run -eq 'y') {
+        dotnet tool update azure.sdk.tools.testproxy --global --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json --version "1.0.0-dev*"
+        return $true
+    } else {
+        return $false
+    }
+}
+
+$run = Read-Host "New test recordings detected.`nWould you like to push them to the assets repository? [y/n]"
```

Comment:
Can we add instructions for how they would do it themselves if they respond with No?

Reply:
I don't think they would need to install the tool, so we can skip that part.